plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -43,7 +43,7 @@
     <consolePatternMatchListener
       class="com.google.cloud.tools.eclipse.ui.util.console.PatternToHyperlinkConverter"
       id="com.google.cloud.tools.eclipse.appengine.deploy.consolePatternMatchListener"
-      regex="https://[a-zA-Z][a-zA-Z0-9-.]*[a-zA-Z0-9]?\.appspot\.com">
+      regex="https://[a-zA-Z0-9][a-zA-Z0-9-.]*[a-zA-Z0-9]\.appspot\.com">
```

Comment:
It might be worth pulling out a small fix like this into a separate PR so it can be quickly reviewed and committed.